views/event.hbs

```diff
@@ -0,0 +1,46 @@
+<ul class="specific-place">
```

Comment:
Could you space out the elements in this file a bit? Kinda hard to read. Also quite a few indentation errors yay :v: